addons/repair/models/repair.py

```diff
@@ -74,7 +76,8 @@ class Repair(models.Model):
     guarantee_limit = fields.Date('Warranty Expiration', states={'confirmed': [('readonly', True)]})
     operations = fields.One2many(
         'repair.line', 'repair_id', 'Parts',
-        copy=True, readonly=True, states={'draft': [('readonly', False)]})
+        copy=True, readonly=True,
+        states={'draft': [('readonly', False)], 'confirmed': [('readonly', False)], 'ready': [('readonly', False)], 'under_repair': [('readonly', False)]})
```

Comment:
What about the `2binvoiced` ? 

Also, maybe inverse the logic, readonly=False (default value) and when state in ('done', 'cancel') => readonly = True. WHat do you think ?

Reply:
Actually for `2binvoice` it depends if the invoice is before or after the repair is done. I'll move the condition in the view to be able to handle this more complex case.